examples/win-title

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Example for $XDG_CONFIG_HOME/nsxiv/exec/win-title
+# Called by nsxiv(1) on each redraw.
+# The output is set as nsxiv's window title.
+# Arguments, "Optional" arguments might be empty:
+#   $1: full path to the current image
+#   $2: current file number
+#   $3: total file number
```

Comment:
```suggestion
#   $1: absolute path of the current image
#   $2: current image number
#   $3: total image count
```

Reply:
`absolute` may not be the best word here since afaik that word is used for fullpath *without* resolving symlinks. But since we use `realpath()` in nsxiv, we are resolving symlinks as well.

Reply:
From `realpath(3)`:
> realpath - return the canonicalized absolute pathname

Maybe solve the confusion altogether and call it "canonicalized absolute pathname", "resolved path" or something similar?

Reply:
"resolved absolute path" seems good to me.